src/ng/directive/form.js

```diff
@@ -295,10 +295,17 @@ function FormController(element, attrs, $scope, $animate, $interpolate) {
    * @description
    * Sets the form to its submitted state.
    */
-  form.$setSubmitted = function() {
+  form.$setSubmitted = function(setOnChildren) {
```

Comment:
I'd name it something more descriptive (e.g. `propagateToChildrenOnly` or `skipPropagationToParent` or something).
(`setOnChildren` made me think we'll always update the parent and we're wondering about the children.)

We should also improve the description, mentioning that it will propagate to parent and children.


Should the arg be public (aka documented).